public/scripts/maps.js

```diff
@@ -94,6 +94,23 @@ $(document).ready(function() {
     }
     renderSVG(mobile, data2.text, false);
   });
+
+  $( "#registerBeacon" ).submit(function( event ) {
+    console.log($('#major').val());
+    console.log($('#minor').val());
+    console.log($('#bType').val());
+    console.log($('#macAddress').val());
```

Comment:
So over here you will use something like `$(this).serializeArray()` to generate an array of objects with keys and values. You'll have to parse that and turn it into an object like this to send to the api.
```
{
    beacon_id: 'A2:21:21:21',
    major: 111,
    minor: 201,
    x: 23.2,
    y: 22,
    building_id: 4
}
```